src/bookshelf/rebar.config

```diff
@@ -13,7 +13,7 @@
     {cf, "",
         {git, "git://github.com/project-fifo/cf.git",                 {branch, "master"}}},
     {chef_secrets, ".*",
-        {git, "git://github.com/chef/chef_secrets.git",               {branch, "master"}}},
+        {git, "git://github.com/chef/chef_secrets.git",               {branch, "lbaker/ej"}}},
```

Comment:
it would be nice to merge whatever is outstanding in lbaker/ej and have this point to master. We could create a ticket and do this as a part of a separate PR.

Reply:
https://github.com/chef/ej/pull/1